blockchain/vm/interpreter.go

```diff
@@ -360,3 +325,12 @@ func (in *EVMInterpreter) Run(contract *Contract, input []byte) (ret []byte, err
 
 	return res, err
 }
+
+func PrintOpCodeExecTime() {
```

Comment:
How about to write a file rather pushing to log in order to easily compile the experiment logs?

Reply:
Thanks for the suggestion but it does not need to print the log periodically.